.github/workflows/update-healthcheck.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow is triggered from developer-portal and updates the healthcheck status in the PR.
+name: Update healthcheck
```

Comment:
Here the same as above. I would rename this to something like "Update developer portal preview"